e2e_samples/parking_sensors_synapse/synapse/workspace/notebook/01_explore.json

```diff
@@ -0,0 +1,59 @@
+{
+	"name": "01_explore",
+	"properties": {
+		"nbformat": 4,
+		"nbformat_minor": 2,
+		"bigDataPool": {
+			"referenceName": "synspdevlace2",
+			"type": "BigDataPoolReference"
+		},
+		"sessionProperties": {
+			"driverMemory": "28g",
+			"driverCores": 4,
+			"executorMemory": "28g",
+			"executorCores": 4,
+			"numExecutors": 2
+		},
+		"metadata": {
+			"kernelspec": {
+				"name": "synapse_pyspark",
+				"display_name": "Synapse PySpark"
+			},
+			"language_info": {
+				"name": "python"
+			}
+		},
+		"cells": [
+			{
+				"cell_type": "code",
+				"metadata": {
+					"collapsed": false
+				},
+				"source": [
+					"import os\r\n",
+					"import datetime\r\n",
+					"\r\n",
+					"# For testing\r\n",
+					"base_path = 'abfss://datalake@mdwdostordevi6c0n.dfs.core.windows.net/data/lnd/2021_08_04_01_28_02'\r\n",
```

Comment:
Is this container intended to be parameterized? It does look like reference data, but want to confirm with you

Reply:
Fixed :) This should basically be replaced by the one running it. Added placeholder comment so its obvious.